.circleci/scripts/binary_populate_env.sh

```diff
@@ -75,8 +75,8 @@ TRITON_VERSION=$(cat $PYTORCH_ROOT/.ci/docker/triton_version.txt)
 # Here PYTORCH_EXTRA_INSTALL_REQUIREMENTS is already set for the all the wheel builds hence append TRITON_CONSTRAINT
 TRITON_CONSTRAINT="platform_system == 'Linux' and platform_machine == 'x86_64'"
 
-# CUDA 12.8 builds have triton for Linux and Linux aarch64 binaries.
-if [[ "$DESIRED_CUDA" == cu128 ]]; then
```

Comment:
at this point can't retrieve this via `` .github/scripts/get_ci_variable.py`` on windows and aarch64 instances when this script is run. Python is not installed yet. Will propose BE PR to move config CUDA_STABLE, PYTORCH_EXTRA_INSTALL_REQUIREMENTS etc... from python to json. This way we can easily consume it via bash or python.